src/css/components/_icons.scss

```diff
@@ -1,3 +1,9 @@
+.dark {
+  .scylla-icon{
+    filter: brightness(0) saturate(100%) invert(100%) sepia(4%) saturate(10%) hue-rotate(140deg) brightness(106%) contrast(100%);
+  }
+}
+
 .scylla-icon {
```

Comment:
Looks safe to remove part of these icons if we do not use them anymore and they are defined in the new icons font.

Reply:
I wanted to wait untill tested in other docs